games/TheBattleForWesnoth/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM  --platform=$TARGETOS/$TARGETARCH ubuntu:22.04
+
+LABEL author="Michael Parker" maintainer="parker@pterodactyl.io"
+
+ENV   DEBIAN_FRONTEND noninteractive
+
+## add container user
+RUN   useradd -m -d /home/container -s /bin/bash container
```

Comment:
since you are need a user, is this not the correct way?

```
## Setup user and working directory
RUN         useradd -m -d /home/container -s /bin/bash container
USER        container
ENV         USER=container HOME=/home/container
WORKDIR     /home/container
```

Reply:
Sorry I am new to making containers. So 
`USER        container`
this sets what user is logged into right?

And 
`ENV         USER=container HOME=/home/container`
These are variables required for pterodactyl to do its thing?

Reply:
> Sorry I am new to making containers. So `USER container` this sets what user is logged into right?
> 
> And `ENV USER=container HOME=/home/container` These are variables required for pterodactyl to do its thing?

no, you only set the user for the container and the working dir.

Reply:
> Got it. That looking better?

you deleted the wronge line :)